models/customer.js

```diff
@@ -0,0 +1,80 @@
+var app = require("../app");
+var db = app.get("db");
+
+var Customer = function(customer) {
+  this.id = customer.id
+  this.name = customer.name
+  this.registered_at = customer.registered_at
+  this.address = customer.address
+  this.city = customer.city
+  this.state = customer.state
+  this.postal_code = customer.postal_code
+  this.phone = customer.phone
+  this.account_credit = customer.account_credit
+  this.due_date = customer.due_date
+  this.checkout_date = customer.checkout_date
+  this.title = customer.title
+}
+module.exports = Customer;
+
+var Movie = require("../models/movie");
+
+// takes on parameter(callback)-then run db.accounts.find
+Customer.all = function(callback) {
```

Comment:
I think these were comments from your accounts model that you also brought over here? Confused me for a while.